src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/TestHostHelper.cs

```diff
@@ -67,11 +69,23 @@ internal static class TestHostHelper
                     outputHelper.WriteLine("End Configuration");
 
                     builder.AddInMemoryCollection(configurationValues);
+
+                    builder.AddInMemoryCollection(new Dictionary<string, string>
```

Comment:
Let's convert `DiagnosticsMonitorCommandHandler.ConfigureStorageDefaults` to an extension method in the `ConfigurationBuilderExtensions` class rather than copying the code here.